README.md

```diff
@@ -71,8 +72,9 @@ In addition to the provided unit tests, we provided a demo game application that
 You can start the demo game application with the following command:
 
 ```bash
-$ yarn run demo-game
+$ npm run demo-game
 ```
+(Note: This command **must** include `run` to in order to work since it is a customize script.)
 
 This will start the Adagrams prompt, and you can start a new game by typing `start` (or `start <num>` for a game with multiple players).
```

Comment:
Should we include a note that control-C can be used to quit (rather than needing to enter `exit` a few times)?

Reply:
Sure, I wrote this: This will start the Adagrams prompt, and you can start a new game by typing `start` (or `start <num>` for a game with multiple players). **To exit the game you can either enter `exit` (you will have to enter this a few times) or press `Control + C` twice.**